digits/extensions/view/boundingBox/config_template.html

```diff
@@ -5,18 +5,3 @@
 {% from "helper.html" import mark_errors %}
 
 <small>Draw a bounding box around a detected object. This expected network output is a nested list of list of box coordinates</small>
```

Comment:
if you don't mind fixing this typo: "This expected... " -> "The expected ..." (and maybe "list of list" -> "list of lists" too). Or if you want to completely rephrase it so it does sound English, you're most welcome to do so :-)